eox_core/settings/test.py

```diff
@@ -58,3 +60,27 @@ def plugin_settings(settings):  # pylint: disable=function-redefined
 
 USERNAME_REGEX_PARTIAL = r'[\w .@_+-]+'
 USERNAME_PATTERN = r'(?P<username>{regex})'.format(regex=USERNAME_REGEX_PARTIAL)
+
+PROCTORING_SETTINGS = {
```

Comment:
Why is this added in this PR?

Reply:
the version 1.5.7 tries to import  signals and signals.py tries to import the constants https://github.com/edx/edx-proctoring/blob/1.5.7/edx_proctoring/apps.py#L117,  the previous version was 1.4.0, but that version is not the platform version.